src/compiler/performanceCore.ts

```diff
@@ -116,6 +98,4 @@ export function tryGetNativePerformanceHooks() {
  *
  * @internal
  */
-export const timestamp = nativePerformance ? () => nativePerformance.now() :
-    Date.now ? Date.now :
-    () => +(new Date());
+export const timestamp = nativePerformanceTime ? () => nativePerformanceTime.now() : Date.now;

```

Comment:
FWIW this must be a closure because Node 19+ require `this` to be correct when calling, so we can't just rip off a method.